channels/viber/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+import plugins.publish.POM_DESCRIPTION
+import plugins.publish.POM_NAME
+
+ext[POM_NAME] = "JAICF-Kotlin Viber Channel"
+ext[POM_DESCRIPTION] = "JAICF-Kotlin Viber Channel implementation. Enables JAICF-Kotlin integration with Viber"
+
+plugins {
+    `jaicf-kotlin`
+    `jaicf-publish`
+}
+
+dependencies {
+    core()
+    api(`coroutines-core`())
+    implementation(ktor("ktor-client-cio"))
```

Comment:
I think these dependencies should be API level

Reply:
Ok